Calendar/View Controllers/DetailsViewController.m

```diff
@@ -0,0 +1,116 @@
+//
+//  DetailsViewController.m
+//  Calendar
+//
+//  Created by Angelina Zhang on 7/13/22.
+//
+
+#import "DetailsViewController.h"
+#import "DetailsView.h"
+#import "ComposeViewController.h"
+
+@interface DetailsViewController () <DetailsViewDelegate, ComposeViewControllerDelegate>
+
+@property (strong, nonatomic) IBOutlet DetailsView *detailsView;
+
+@end
+
+@implementation DetailsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.detailsView.delegate = self;
+    [self updateDetailsView];
+}
+
+- (void)updateDetailsView {
+    self.detailsView.eventTitleLabel.text = self.event.eventTitle;
+    self.detailsView.timeLabel.text = [self getTimeString];
+    
+    if ([self.event.location isEqual:@""]) {
+        [self.detailsView.locationIcon setHidden:true];
+        [self.detailsView.locationLabel setHidden:true];
+        
+        [self.detailsView.descriptionLocationTopConstraint setConstant:0];
```

Comment:
This is easy enough to reason about but it might be nice to have a `displayLocation` helper method.